### ISSUE TITLE
rustdesk@1.1.9: hash check failed

### ISSUE THREAD

**Issue by ShawnTheShadow** [ScoopInstaller/Extras#8722]
#### Scoop Configuration

```json
{
  "lastupdate": "2022-06-19T23:54:54.4197459-05:00",
  "SCOOP_REPO": "https://github.com/lukesampson/scoop",
  "SCOOP_BRANCH": "master"
}
```
```
WARN  Purging previous failed installation of rustdesk.
ERROR 'rustdesk' isn't installed correctly.
Removing older version (1.1.9).
'rustdesk' was uninstalled.
Installing 'rustdesk' (1.1.9) [64bit]
rustdesk-1.1.9-windows_x64.zip (6.3 MB) [=====================================================================] 100%
Checking hash of rustdesk-1.1.9-windows_x64.zip ... ERROR Hash check failed!
App:         extras/rustdesk
URL:         https://github.com/rustdesk/rustdesk/releases/download/1.1.9/rustdesk-1.1.9-windows_x64.zip
First bytes: 50 4B 03 04 14 00 00 00
Expected:    3d6bed7c514dae3b4a8d82c5d68c19983a355b54a8d7e788aada9a0f3e557cbc
Actual:      c6fc4d9099e38a654fa364cf57c6128eb4d6186d7444a5d0b3a714b58d0641b7
````

**Comment by codekoala** [comment]
This appears to be a problem again, for me anyway:
![image](https://user-images.githubusercontent.com/48698/180135279-7f7b3fa6-54e0-462a-8b78-290d5244b980.png)

**Comment by e12e** [comment]
Yup, appears to be (still?) broken:

```
scoop install rustdesk
WARN  Purging previous failed installation of rustdesk.
ERROR 'rustdesk' isn't installed correctly.
Removing older version (1.1.9).
'rustdesk' was uninstalled.
Installing 'rustdesk' (1.1.9) [64bit]
rustdesk-1.1.9-windows_x64.zip (6,3 MB) [=====================================================================] 100%
Checking hash of rustdesk-1.1.9-windows_x64.zip ... ERROR Hash check failed!
App:         extras/rustdesk
URL:         https://github.com/rustdesk/rustdesk/releases/download/1.1.9/rustdesk-1.1.9-windows_x64.zip
First bytes: 50 4B 03 04 14 00 00 00
Expected:    c6fc4d9099e38a654fa364cf57c6128eb4d6186d7444a5d0b3a714b58d0641b7
Actual:      a5e857af9424f628cbad72b6b60bba43c5a66a225b493d4573276debf0074496

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/lukesampson/scoop-extras/issues/new?title=rustdesk%401.1.9%3a+hash+check+failed
```

**Comment by e12e** [comment]
Looks like this might actually be an upstream issue, judging from the pr that fixed this previously - the "same" (1.1.9) build seem to keep changing?
